framework/src/main/java/io/github/kgress/scaffold/WebDriverManager.java

```diff
@@ -588,6 +588,8 @@ private RemoteWebDriver configureSauceRemoteBrowser(MutableCapabilities browserO
       sauceCaps.setCapability("username", username);
       sauceCaps.setCapability("accessKey", accessKey);
       sauceCaps.setCapability("name", testName);
+      sauceCaps.setCapability("extendedDebugging", sauce.isExtendedDebugging());
+      sauceCaps.setCapability("capturePerformance", sauce.isCapturePerformance());
```

Comment:
Comment on `DesiredCapabiliitiesConfigurationProperties` will impact this addition slightly, moving it to just under line 610 with the `Optional`'s `.isPresent()` check.